shuffle/cli/tests/common/mod.rs

```diff
@@ -82,7 +110,8 @@ impl ShuffleTestHelper {
             &self.home,
             new::DEFAULT_BLOCKCHAIN.to_string(),
             self.project_path(),
-        )
+        )?;
+        self.overwrite_networks_config_into_toml()
```

Comment:
network configuration lives outside (~/.shuffle) the scope of project (/tmp/someproject) so i would put this in `new`, where you create the `networks_config`.

Reply:
will place this in `new`